VideoLocker/src/main/java/org/edx/mobile/view/adapters/MyCourseAdapter.java

```diff
@@ -187,32 +183,29 @@ public BaseViewHolder getTag(View convertView) {
                 .findViewById(R.id.school_code_tv);
         holder.starting_from = (TextView) convertView
                 .findViewById(R.id.starting_from);
-        holder.courseImage = (NetworkImageView) convertView
+        holder.courseImage = (ImageView) convertView
                 .findViewById(R.id.course_image);
-        holder.new_course_content = (LinearLayout) convertView
+        holder.newCourseContent = (ViewGroup) convertView
                 .findViewById(R.id.new_course_content_layout);
-        holder.starting_from_layout = (LinearLayout) convertView
-                .findViewById(R.id.starting_from_layout);
-        holder.certificateBanner = (RelativeLayout) convertView
+        holder.certificateBanner = convertView
                 .findViewById(R.id.course_certified_banner);
         holder.facePileView = (SocialFacePileView) convertView
                 .findViewById(R.id.course_item_facepileview);
-        holder.facePileContainer = (LinearLayout) convertView
+        holder.facePileContainer = (ViewGroup) convertView
                 .findViewById(R.id.course_item_facepile_container);
         return holder;
     }
 
     private static class ViewHolder extends BaseViewHolder {
-        NetworkImageView courseImage;
+        ImageView courseImage;
         TextView courseTitle;
         TextView schoolCodeTv;
         TextView orgCodeTv;
         TextView orSymbolTv;
         TextView starting_from;
-        LinearLayout new_course_content;
-        LinearLayout starting_from_layout;
-        RelativeLayout certificateBanner;
-        LinearLayout facePileContainer;
+        View certificateBanner;
+        ViewGroup newCourseContent;
```

Comment:
I think we can also change the type of this to just `View`, as there seems to be no inherent necessity for it to be a container.